tests/tacacs/utils.py

```diff
@@ -45,12 +45,14 @@ def setup_tacacs_client(duthost, tacacs_creds, tacacs_server_ip):
     """setup tacacs client"""
 
     # configure tacacs client
+    default_tacacs_server = []
```

Comment:
If it is a list, it would be better to use plural for variable name, like `default_tacacs_servers`.